### DISCUSSION TITLE
Add possibility to restart a single instance

### DISCUSSION THREAD

**Discussion by cybergogo** [paukstelis/octoprint_deploy#155] [Ideas]
Hello, 
I'm running 6 instances for my prusa printers. 
Once in a while, one of the instances fails and the only choice we have is to restart all instances. Would it be possible to restart only one please ? Because it's pretty annoying when part of the system is down and you can't restart all of them because you have to wait for a 24h print to finish first 😢 
Thank you so much.
Best regards.

**Comment by paukstelis** [comment]
You can already restart an instance any time you want through the Web UI. Just go to the power button in the top bar and `Restart OctoPrint`. You can also restart any instance via the commandline whenever you need to: `sudo systemctl restart instancename`

**Reply by cybergogo** [reply]
Wow, thank you. I'm still a bit noob regarding CLI. 🙌
